src/software/jetson_nano/ansible/playbooks/setup_systemd.yml

```diff
@@ -0,0 +1,71 @@
+---
+
+- name: Setting up systemd
+  hosts: THUNDERBOTS_HOSTS
+
+  tasks:
+
+    - debug:
+        msg: "[Robot ID = {{ inventory_hostname }}]"
+
+    - name: sync binaries to the nano
+      with_items:
+        - ../../../../../../../../jetson_nano/display
+        - ../../../../../../../../jetson_nano/broadcasts
+        - ../../../../../../../../jetson_nano/thunderloop_main
+        - ../../../../../../../../jetson_nano/redis
```

Comment:
do we need these? seems to work well without

Reply:
it shouldnt :o unless you had the binaries on the robots already?